### ISSUE TITLE
Provide some Cypher queries to know the details about the KG

### ISSUE THREAD

**Issue by pkumar81** [unmtransinfo/cfde-distillery#6]
Add some sample Cypher queries to the repository to learn basic information about the knowledge graph.

**Comment by pkumar81** [comment]
Sample Cypher queries were written and tested on the graph database. Queries are in the cql folder: https://github.com/unmtransinfo/cfde-distillery/tree/main/cql 

Disease and drug nodes need to be added to the graph database for more sample queries for use cases.

**Comment by vmetzger09** [comment]
I agree Praveen.  As Cristian pointed out on Monday morning, we really need those disease and drug nodes added to the KG before it is possible to construct queries that are adequately complex to serve as a stepping-stone to use-case development.

**Comment by pkumar81** [comment]
The latest KG contains Diseases, Compounds, and Proteins.
